gateway/src/main.rs

```diff
@@ -37,6 +38,20 @@ async fn main() -> io::Result<()> {
         Sqlite::create_database(db_uri).await.unwrap();
     }
 
+    let docker_stats_path =
+        PathBuf::from_str(DOCKER_STATS_PATH).expect("to parse docker stats path");
+
+    // Return an error early if the docker stats path is not in the expected location.
+    if !docker_stats_path.exists() {
+        return Err(std::io::Error::new(
+            io::ErrorKind::NotFound,
+            format!(
+                "could not find docker stats at path: {:?}",
+                DOCKER_STATS_PATH
+            ),
+        ));
+    }
```

Comment:
We should just ensure this works for local development, then we can merge it. I don't imagine it will work on Windows, but developing Shuttle locally on Windows is already limited. For Linux, we need to see which version of cgroups are available, or just check if either path exists. For mac, I'm not sure, wdyt @iulianbarbu? See the `get_container_stats` function for which paths we use on linux.

Reply:
We'll release this without the local development fix, we can do that after.